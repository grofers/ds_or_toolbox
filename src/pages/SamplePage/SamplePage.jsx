import React from "react";
import SamplePageHeader from "./components/SamplePageHeader";
import SamplePageForm from "./components/SamplePageForm";
import SamplePageTable from "./components/SamplePageTable";

const SamplePage = () => (
  <main>
    <SamplePageHeader />
    <SamplePageForm />
    <SamplePageTable />
  </main>
);

export default SamplePage;
