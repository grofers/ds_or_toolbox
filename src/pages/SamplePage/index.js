export { default } from "./SamplePage.jsx";
