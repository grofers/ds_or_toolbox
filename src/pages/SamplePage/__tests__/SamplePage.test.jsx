import React from "react";
import { render } from "@testing-library/react";
import SamplePage from "../SamplePage";

describe("Sample Page", () => {
  it("should match the snapshot on page load", () => {
    const { asFragment } = render(<SamplePage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
