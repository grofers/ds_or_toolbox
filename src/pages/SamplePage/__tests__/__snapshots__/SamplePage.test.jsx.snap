// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample Page should match the snapshot on page load 1`] = `
<DocumentFragment>
  <main>
    <section>
      Sample Header
    </section>
    <section>
      Sample Form
    </section>
    <section>
      Sample Table
    </section>
  </main>
</DocumentFragment>
`;
